.github/workflows/dotnetcore.yml

```diff
@@ -1,5 +1,6 @@
 name: "Run unit tests"
 
+permissions: write-all
```

Comment:
Is it possible to reduce the scope of this permission?

Reply:
done